src/Sarif/CodeGenHints.json

```diff
@@ -114,8 +114,8 @@
       "arguments": {
         "typeName": "AlgorithmKind",
         "description": "Values specifying different hashing algorithms.",
-        "zeroValue": "Unknown",
-        "enumValues": [
+        "zeroValueName": "Unknown",
+        "enumNames": [
```

Comment:
'memberNames' and 'memberValues' might have been clearer. take this comment or leave it. :)

---
In reply to: [62882321](https://github.com/Microsoft/sarif-sdk/pull/259#discussion_r62882321) [](ancestors = 62882321)